plugin/db/mysql/restore.go

```diff
@@ -250,7 +249,7 @@ func GetBinlogReplayList(startBinlogInfo api.BinlogInfo, binlogDir string) ([]st
 		return nil, errors.Wrapf(err, "cannot parse the start binlog file name %q", startBinlogInfo.FileName)
 	}
 
-	binlogFiles, err := ioutil.ReadDir(binlogDir)
+	binlogFiles, err := os.ReadDir(binlogDir)
```

Comment:
Prefer to rename the first result of `os.ReadDir` to `xxxDirEntry`.

Reply:
Short names are probably okay. We don't need to mention the types.

Reply:
SG.